transport/messages/result_message_base.hh

```diff
@@ -9,16 +9,20 @@
 
 #pragma once
 
+#include <map>
 #include <vector>
 #include <seastar/core/sstring.hh>
 
 #include "seastarx.hh"
+#include "locator/tablets.hh"
+#include "replica/tablets.hh"
 
 namespace cql_transport {
 namespace messages {
 
 class result_message {
     std::vector<sstring> _warnings;
+    std::unordered_map<sstring, bytes> _custom_payload;
 public:
```

Comment:
default ctor of unordered map allocates IIRC, better to use optional so we won't penalize requests that don't need a custom payload.

Reply:
This should be apparent in perf's allocation report.